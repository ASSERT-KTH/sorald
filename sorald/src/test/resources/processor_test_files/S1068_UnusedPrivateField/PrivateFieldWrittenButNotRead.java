public class PrivateFieldWrittenButNotRead {
    private int x; // Noncompliant

    private String foo; // Noncompliant

    private long used;

    public PrivateFieldWrittenButNotRead(int x, int y) {
        this.x = x + y;
    }

    public void initialize(String bar) {
        foo = bar;
        used = 234324L;
    }

    public long getUsed() {
        return used;
    }
}
