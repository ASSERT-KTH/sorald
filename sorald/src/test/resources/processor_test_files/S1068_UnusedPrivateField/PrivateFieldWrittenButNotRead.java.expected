public class PrivateFieldWrittenButNotRead {
    private long used;

    public PrivateFieldWrittenButNotRead(int x, int y) { }

    public void initialize(String bar) {
        used = 234324L;
    }

    public long getUsed() {
        return used;
    }
}